### PR TITLE
Rewrite user input part

### PR DESCRIPTION
**Pull request by locomarco**

### CODE REVIEW

File: firmware/EsaSpeedSwitch/EsaSpeedSwitch.ino [hunk 1]
```diff
@@ -24,229 +24,238 @@
...
-    delay(100);
+
+    if (i < (count - 1))
+    {
+      delay(pause);
+    }
   }
 }
 
+void blinkEco()
```
Comment (ands) [inline firmware/EsaSpeedSwitch/EsaSpeedSwitch.ino:185]: I like the idea of using the eco led as a feedback channel :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+It makes use of throttle & brake levers and the button on the dashboard.
+
+## How to use?
+
+There are 3 speed settings LOW (20 km/h), MEDIUM (25 km/h) and HIGH (30 km/h) and the option to enable/disable lock mode.
+If you use the original hardware, the max. speed after power on is always 20 km/h, so you have to set a new speed every time.
+A successful change of speed is shown by switching to eco mode for a short time (green LED lights up).
+You can not change speed limit of eco mode, it will stay at 15 km/h.
+
+- 3x button or 5x button = 20 km/h
```
Comment (ands) [inline README.md]: I'd prefer if there was still a way to reset the speed limit by pulling the brake for a few seconds or so. Would you mind adding that? :) The button press codes could still remain this way.

**Review by ands — APPROVED** [review]

File: firmware/EsaSpeedSwitch/EsaSpeedSwitch.ino (hunk 1 above)
Comment (locomarco) [reply firmware/EsaSpeedSwitch/EsaSpeedSwitch.ino:185]: I also tried using the speedometer itself, but that didn't work... Yet 🤓

File: README.md (hunk 2 above)
Comment (locomarco) [reply README.md]: I thought about that, too.
We should have an option to disable tuning very inconspicuous, but without doing it accidentally.
I'll add 5 seconds full brake or something like that.

Comment (locomarco) [reply README.md]: I just added an option to reset max speed by holding down throttle and brake for 3 seconds.
This should be fast and easy enough, no one becomes suspicious and it shouldn't happen by accident in normal driving.

I guess we could even remove the other options of detuning (3x / 5x button).

Comment (ands) [reply README.md]: I don't see that change here. Will it be in a separate pull request?

Comment (locomarco) [reply README.md]: Sorry, i didn't push it yet... 😅

**Pull request merged**